src/main/java/org/terasology/metalrenegades/ai/component/NeedsComponent.java

```diff
@@ -28,23 +33,7 @@
 public class NeedsComponent implements Component {
 
     @Replicate
-    public CitizenNeed hungerNeed;
-
-    @Replicate
-    public CitizenNeed thirstNeed;
-
-    @Replicate
-    public CitizenNeed socialNeed;
-
-    @Replicate
-    public CitizenNeed restNeed;
-
-    public NeedsComponent(CitizenNeed hunger, CitizenNeed thirst, CitizenNeed social, CitizenNeed rest) {
-        this.hungerNeed = hunger;
-        this.thirstNeed = thirst;
-        this.socialNeed = social;
-        this.restNeed = rest;
-    }
+    public List<CitizenNeed> needs = Lists.newArrayList();;
 
     public NeedsComponent() {
```

Comment:
If this constructor does nothing then it's probably cleaner to remove it and use Java's default

Reply:
Agreed, the extra constructor has been removed 👍